test/integration_suite_test.go

```diff
@@ -9,11 +9,15 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 	"testing"
 
+	"gopkg.in/yaml.v3"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	"github.com/rwx-research/captain-cli/internal/cli"
```

Comment:
💭 I'm a little worried that this too-loosely couples the integration tests with the format of the config file. If format changes, the integration tests may still pass.

Reply:
should be handled by "backwards compatibility" tests but makes it difficult to know which tests to run. Let's discuss this in person